src/data/userName.js

```diff
@@ -0,0 +1 @@
+export const userName = 'Voyager';
```

Comment:
if it looks like a constant, behaves like a constant, used like a constant, it should follow our constants naming convention.

+ name `userName` is too abstract

```suggestion
export const DEFAULT_USERNAME = 'Voyager';
```

or 
```suggestion
export const SYSTEM_USERNAME = 'Voyager';
```

Reply:
not clever enough